backend/src/services/objective.service.ts

```diff
@@ -0,0 +1,74 @@
+import { getCustomRepository } from 'typeorm';
+import { Objective } from '~/data/entities/objective';
+import { OKR } from '~/data/entities/okr';
+import Okrepository from '~/data/repositories/okr.repository';
+import ObjectiveRepository from '~/data/repositories/objective.repository';
+import { badRequestError } from '~/common/errors';
+
+export const createObjectiveToOkr = async ({
+  okrId,
+  body,
+}: {
+  okrId: string;
+  body: Objective;
+}): Promise<OKR> => {
+  if (!body.name) throw badRequestError('Objective name is undefined!!!');
+
+  const okrRepository = getCustomRepository(Okrepository);
+  const objectiveRepository = getCustomRepository(ObjectiveRepository);
+
+  const okr = await okrRepository.findOne({ id: okrId });
+  const isObjectiveExist = await objectiveRepository.findOne({
+    name: body.name,
+  });
+
+  if (isObjectiveExist)
+    badRequestError(`Objective with name ${body.name} is exist!!!`);
```

Comment:
Use brackets even it is single line condition
```suggestion
  if (isObjectiveExist) {
    badRequestError(`Objective with name ${body.name} is exist!!!`);
  }  
```